RELEASE-NOTES.md

```diff
@@ -1,3 +1,117 @@
+# PatternFly-React Albatross (2019-07-02)
+These releases are part of the PatternFly 4 - Albatross milestone:
+ - @patternfly/react-core@3.58.1
+ - @patternfly/react-inline-edit-extension@2.9.12
+ - @patternfly/react-table@2.13.43
+ - @patternfly/react-topology@2.4.21
+ - @patternfly/react-virtualized-extension@1.1.45
+ - @patternfly/react-charts@4.4.7
+ - @patternfly/react-styles@3.4.6
+ - @patternfly/react-tokens@2.6.5
+
+**Charts (@patternfly/react-charts@4.4.7):**
+*   Added sparkline example and interpolation to area examples ([#2373](https://github.com/patternfly/patternfly-react/pull/2373))
+*   Fixed themes for bar chart default colors ([#2112)](https://github.com/patternfly/patternfly-react/pull/2112)
+*   DonutThreshold): No longer show static threshold donut tooltips by default.([#2270](https://github.com/patternfly/patternfly-react/pull/2270))
+*   Added support for legends and grids ([#2324](https://github.com/patternfly/patternfly-react/pull/2324))
+
+**Components (@patternfly/react-core@3.58.1):**
+*   **About modal:** changed strapline element to paragraph ([#2376](https://github.com/patternfly/patternfly-react/pull/2376))
+*   **Breadcrumb switcher:** Disabled autocomplete ([#2407](https://github.com/patternfly/patternfly-react/pull/2407))
+*   **Card:** Add compacted card variation ([#2399](https://github.com/patternfly/patternfly-react/pull/2399))
+*   **Checkbox:** Added 3rd state to checkbox controlled by consumer ([#2252](https://github.com/patternfly/patternfly-react/pull/2252))
+*   **Chip group:** Added heading level prop to toolbar chip group label ([#2278](https://github.com/patternfly/patternfly-react/pull/2278))
+*   **Data list:** Added DataListRow to imports ([#2353](https://github.com/patternfly/patternfly-react/pull/2353))
+*   **Dropdown:** Automatically focused first dropdown item when DropdownMenu ([#2355](https://github.com/patternfly/patternfly-react/pull/2355))
+*   **Expandable:** Added button type to the expandable component ([#2340](https://github.com/patternfly/patternfly-react/pull/2340))
+*   **Input group:** Convert input group to typescript ([#2220](https://github.com/patternfly/patternfly-react/pull/2220))
+*   **Label:** added typescript flag to the label docs ([#2343](https://github.com/patternfly/patternfly-react/pull/2343))
```

Comment:
This should be TypeScript?

Reply:
@tlabaj  - I don't understand - these are component changes, not typescript conversions - wh not put them with component changes?